thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/controller.js

```diff
@@ -49,6 +49,11 @@ export default Controller.extend({
     resolution: 'anomalyFeedback'
   },
 
+  /**
+   * Make duration service accessible
+   */
+  durationCache: service('services/duration'),
```

Comment:
Can we start with the convention custom services we built ends with `Service` and `ApiService` if it's for api?

example
```
  anomaliesApiService: service('services/api/anomalies'),
```

Reply:
Sounds good. However, this one is not related to API - no fetching of data from server. It only caches user-selected time ranges for various routes to read/update.